src/main/java/de/tum/in/www1/artemis/service/notifications/NotificationSettingsService.java

```diff
@@ -127,9 +127,9 @@ public class NotificationSettingsService {
 
     // This set has to equal the UI configuration in the client notification settings structure file!
```

Comment:
This comment states, that you also have to change things in the client, but I cannot see any according changes

Reply:
I removed the constant